src/libstore/store-api.hh

```diff
@@ -838,6 +839,9 @@ std::optional<ValidPathInfo> decodeValidPathInfo(
     std::istream & str,
     bool hashGiven = false);
 
+/* Compute the prefix to the hash algorithm which indicates how the files were
+   ingested. */
+std::string makeFileIngestionPrefix(const FileIngestionMethod m);
```

Comment:
Wrong indentation?

Reply:
Oh, actually this is just a freestanding function and the closing brace is the namespace (so no indent). I guess in the later PR it gets a better home in `content-address.h`.